### PR TITLE
feat: unify side effects between sdk and circuit 

### PR DESCRIPTION
**Pull request by beqaabu**
## Summary
This pr transforms side effects from `LocalTrigger` to regular side effects that the protocol can validate. 

## Checklist
- [ ] insert any notable changes, in order of commit

## Please check that your PR completes the requirements as follows
- [ ] Tests have been added for bug fixes/features
- [ ] Acceptance criteria from the issue has been completed

### CODE REVIEW

**Review by AwesomeIbex — DISMISSED** [review]
approved since my review is blocking the pr and I fixed the issues

File: pallets/circuit/src/mock.rs [hunk 1]
```diff
@@ -566,7 +566,7 @@ impl ExtBuilder {
                 token_decimals: 10,
             },
             vec![],
-            vec![*b"tran", *b"swap", *b"aliq"],
+            vec![*b"tran", *b"swap", *b"aliq", *b"mult", *b"call"],
```
Comment (MaciejBaj) [inline pallets/circuit/src/mock.rs:569]: re-use from `t3rn_protocol::side_effects::standards::standard_side_effects()`

File: primitives/src/match_format.rs [hunk 2]
```diff
@@ -53,6 +55,17 @@ pub fn trim_whitespace(input_string: StrLike) -> StrLike {
     result
 }
 
+pub fn match_side_effect(kind: &StrLike) -> Result<Bytes, &'static str> {
```
Comment (MaciejBaj) [inline primitives/src/match_format.rs:58]: align and cover all cases from `t3rn_protocol::side_effects::standards::standard_side_effects()`

File: types/src/side_effect.rs [hunk 3]
```diff
@@ -36,6 +39,147 @@ impl<
...
+            4 => Ok(Action::Call),
+            _ => Err("Invalid action id"),
+        }
+    }
+}
+
+impl Into<[u8; 4]> for Action {
+    fn into(self) -> [u8; 4] {
+        match self {
+            Action::Transfer => *b"tran",
```
Comment (MaciejBaj) [inline types/src/side_effect.rs:122]: same, handle cases from `t3rn_protocol::side_effects::standards::standard_side_effects()`

**Review by MaciejBaj — CHANGES_REQUESTED** [review]

**Review by MaciejBaj — APPROVED** [review]

**Pull request merged**